app/utils/get_filtered_records.py

```diff
@@ -206,24 +210,19 @@ def date_filter(record, start_date=None, end_date=None, use_sampling_date=True):
         # If there are multiple coordinates for a given geographic level, use the average
         # Note this needs to happen after prioritization of estimates
         # because the prioritization function may group multiple estimates from the same study into 1
-        region_types = ['country', 'state', 'city']
-        for region_type in region_types:
-            if len(record[f'{region_type}_latitude']) == 0:
-                # Edge case for when there isn't even a country level coordinate
-                if 'pin_latitude' not in record:
-                    record['pin_latitude'] = None
-                    record['pin_longitude'] = None
-                    record['pin_region_type'] = ''
-                break
-            # If we find multiple coordinates for a given geographic level
+        record['pin_latitude'] = record['country_latitude']
+        record['pin_longitude'] = record['country_longitude']
+        record['pin_region_type'] = 'country' if record['country_latitude'] is not None else ''
+        for region_type in ['state', 'city']:
```

Comment:
no more country region type?

Reply:
Lines 213 - 215 set the pin_latitude/longitude to the country latitude/longitude to begin with. Just thought it was cleaner to write it this way, esp because state and city lat/lngs are lists while country lat/lngs are single values.